images/image-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 The Kubernetes Authors.
+# Copyright 2022 The Kubernetes Authors.
```

Comment:
This is just follow up for https://github.com/kubernetes-sigs/cri-tools/pull/1006#discussion_r998240151